tests/Services/SearchServiceTest.php

```diff
@@ -121,6 +127,13 @@ public function setUp()
             ->disableOriginalConstructor()
             ->setMethods([])
             ->getMock();
+        $this->templateConfigService = $this->getMockBuilder(TemplateConfigService::class)
+            ->disableOriginalConstructor()
+            ->setMethods([])
+            ->getMock();
+
+        global $classInstances;
```

Comment:
I read `global` and my brain goes brrr. Sad though that we need to opt for this way, as we're really restricted by PHPUnit and Plentymarkets here.